core/spec_initializer.py

```
@@ -0,0 +1,490 @@
+                {"id": "stage_3", "name": "总结记录", "file": "03_summarize.md"}
+            ]
+        }
+
+        stages_data = pipeline_map.get(task_type.value, [])
+        pipeline = []
+
+        for stage_data in stages_data:
+            stage = PipelineStage(**stage_data)
+            pipeline.append(stage)
+
+            # 创建步骤文件（传递 task 参数）
```

Ruff (0.15.9)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)